ts/components/ui/ListItemTransaction.tsx

```diff
@@ -82,6 +82,20 @@ const LeftComponent = ({ logoIcon }: LeftComponentProps) => {
   return getCardLogoComponent(logoIcon as IOLogoPaymentType, CARD_LOGO_SIZE);
 };
 
+/**
+ * This function is used to get the text that will be read by the screen reader
+ * with the correct minus symbol pronunciation.
+ */
+const getAccessibleAmountText = (amount?: string) =>
+  pipe(
+    amount,
+    O.fromNullable,
+    O.map(amount =>
+      amount.replace("-", I18n.t("global.accessibility.minusSymbol"))
+    ),
+    O.getOrElseW(() => undefined)
+  );
+
```

Comment:
This function could become an utility function, maybe placed in `ts/utils/a11y.ts`. What do you think?

Reply:
Yep, I agree with you!

Addressed in [85815b5](https://github.com/pagopa/io-app/pull/4839/commits/85815b591c104a31d23a24b479e3106a9f141b3a)